test_scripts/API/VehicleData/GetVehicleData/007_GetVD_app_version_is_greater_than_parameter_version.lua

```diff
@@ -0,0 +1,41 @@
+---------------------------------------------------------------------------------------------------
+-- Description: Check that SDL processes vehicle data RPCs with <vd_param> parameter
+-- if an app is registered with version greater than current parameter version
+--
+-- Preconditions:
+-- 1) SDL and HMI are started
+-- 2) Vehicle Data RPCs and <vd_param> parameter are allowed by policies
+-- 3) <vd_param> parameter has param_version = 5.1.0
```

Comment:
Propose to remove version numbers and have a description of case like `App is registered with syncMsgVersion greater than a version of <vd_param> parameter` because different parameters have different versions.